staff/lab/mod-printer

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+"""Remove or add printers to/from CUPS classes."""
+import argparse
+import subprocess
+import sys
+
+
+def modify_printer(args):
+    """Perform internal call to CUPS CLI to modify class(es)."""
+    # Determine which CUPS classes are available
+    cups_classes = subprocess.check_output(["lpstat", "-p"]).decode("utf-8").split("\n")
```

Comment:
Use the more modern `subprocess.run` function here. It'll end up being cleaner since it'll automatically decode the output, you can configure it to throw an exception if there's an error exit code, etc.